.cursor/skills/dev-helper/scripts/_config.sh

```
@@ -0,0 +1,49 @@
+
+CONFIG_FILE="$SKILL_DIR/dev-helper.config.json"
+
+if [[ ! -f "$CONFIG_FILE" ]]; then
+  echo "ERROR: Missing config file: $CONFIG_FILE" >&2
+  echo "Copy dev-helper.config.example.json to dev-helper.config.json and fill in your values." >&2
+  exit 1
+fi
+
+config_get() { jq -r "$1" "$CONFIG_FILE"; }
+
+JIRA_BASE_URL=$(config_get '.jira.baseUrl')
```

Shellcheck (0.11.0)

[warning] JIRA_BASE_URL appears unused. Verify use (or export if used externally).

(SC2034)